### PR TITLE
Create Pull Request Template

### LINKED ISSUE #46: Create a PULL_REQUEST_TEMPLATE
#### Issue Summary

We would like to have a template for describing what has changed in the pull request being submitted.

[Github docs on this](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/) 

You can use [this file](https://github.com/sendgrid/open-source-library-data-collector/blob/master/.github/PULL_REQUEST_TEMPLATE.md) as an example

### PR DESCRIPTION
**Pull request by pawel-lewtak**
Fixes #46

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/php-http-client?pullRequest=47) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/php-http-client?pullRequest=47) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/php-http-client?pullRequest=47) it.</sub>

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+- [ ] Code passes all existing [tests](https://github.com/sendgrid/php-http-client/tree/master/test)
+- [ ] Any new functionality added includes new unit tests in [`tests/unit`](https://github.com/sendgrid/php-http-client/tree/master/test/unit)
+- [ ] Create or update example code to show the new functionality in action.
+- [ ] All code, branch, and git naming and style conventions are followed (see [`CONTRIBUTING.md`](https://github.com/sendgrid/php-http-client/blob/master/CONTRIBUTING.md#style-guidelines-and-naming-conventions))
+- [ ] Feature branch has been rebased off of the latest `master` branch. (see [`CONTRIBUTING.md`](https://github.com/sendgrid/php-http-client/blob/master/CONTRIBUTING.md#creating-a-pull-request)).
+
+If you have questions, please send an email [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.
+
+<!--
```
Comment (mbernier) [inline .github/PULL_REQUEST_TEMPLATE.md]: Can you please remove this? I should have added that to the issue. Sorry!!

Comment (pawel-lewtak) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done. Thanks for pointing that out.

**Comment by thinkingserious** [comment]
Hello @pawel-lewtak,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**